OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -144,57 +144,78 @@ void InverseKinematicsTool::setNull()
+        "constraint errors are appended to the cost function.");
     _constraintWeightProp.setName("constraint_weight");
     _constraintWeightProp.setValue(std::numeric_limits<SimTK::Real>::infinity());
     _propertySet.append( &_constraintWeightProp );
 
-    _accuracyProp.setComment("The accuracy of the solution in absolute terms. I.e. the number of significant"
-        "digits to which the solution can be trusted.");
+    _accuracyProp.setComment(
+        "The accuracy of the solution in absolute terms. Default is 1e-5. "
+        "It determines the number of significant digits to which the solution "
+        "can be trusted.");
     _accuracyProp.setName("accuracy");
     _accuracyProp.setValue(1e-5);
     _propertySet.append( &_accuracyProp );
 
-    _ikTaskSetProp.setComment("Markers and coordinates to be considered (tasks) and their weightings.");
+    _ikTaskSetProp.setComment(
+        "Markers and coordinates to be considered (tasks) and their weightings. "
+        "The sum of weighted-squared task errors composes the cost function.");
     _ikTaskSetProp.setName("IKTaskSet");
     _propertySet.append(&_ikTaskSetProp);
 
-    _markerFileNameProp.setComment("TRC file (.trc) containing the time history of observations of marker positions.");
+    _markerFileNameProp.setComment(
+        "TRC file (.trc) containing the time history of observations of marker "
+        "positions obtained during a motion capture experiment. Markers in this "
+        "file that have a corresponidng task and model marker are included.");
     _markerFileNameProp.setName("marker_file");
     _propertySet.append(&_markerFileNameProp);
 
-    _coordinateFileNameProp.setComment("The name of the storage (.sto or .mot) file containing coordinate observations."
-        "Coordinate values from this file are included if there is a corresponding coordinate task. ");
+    _coordinateFileNameProp.setComment(
+        "The name of the storage (.sto or .mot) file containing the time history"
+        " of coordinate observations. Coordinate values from this file are "
+        "included if there is a corresponding model coordinate and task. ");
     _coordinateFileNameProp.setName("coordinate_file");
     _propertySet.append(&_coordinateFileNameProp);
 
-    const double defaultTimeRange[] = {-std::numeric_limits<SimTK::Real>::infinity(), std::numeric_limits<SimTK::Real>::infinity()};
-    _timeRangeProp.setComment("Time range over which the inverse kinematics problem is solved.");
+    const double defaultTimeRange[] =
+        {-std::numeric_limits<SimTK::Real>::infinity(),
+          std::numeric_limits<SimTK::Real>::infinity()};
+    _timeRangeProp.setComment(
+        "The desired time range over which inverse kinematics is solved. "
+        "The closest start and final times from the provided observations "
+        "are used to specify the actual time range to be processed.");
```

Comment:
@tkuchida I updated the Tool comment to reflect that the specified time range is "desired" and how the actual time range is determined.